scripts/build/build.js

```diff
@@ -99,7 +99,7 @@ async function run(params) {
     await execa("rm", ["-rf", ".cache"]);
   }
 
-  const bundleCache = new Cache(".cache/", "v18");
+  const bundleCache = new Cache(".cache/", "v19");
```

Comment:
Is this related to the issue or something else you found?

Reply:
Cache version should always update, when updating build script, I've update this many times.

Reply:
I's because build script changes is not calculated in cache, only effect people who run build, like you, haha